src/components/Plans.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect } from "react";
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { FaEdit } from "react-icons/fa";
+import { MdAdd } from "react-icons/md";
+import PlansModal from "./modals/PlansModal";
+
+function Plans() {
+    const [plans, setPlans] = useState([]);
+    const [openModal, setOpenModal] = useState(false);
+
+
+
+    useEffect(() => {
+        getPlansDetails();
+    }, [])
+    const getPlansDetails = async () => {
+        if (!openModal) {
+            let plans = await fetch('http://3.111.147.217:3000/plans');
+            plans = await plans.json();
+            setPlans(plans);
+        }
+    }
+
+    console.log("Plans", plans)
+    return (
+        <div className="batch-list">
+            <h3 className="batch-heading">Plans <button onClick={() => { setOpenModal(true) }}>{<MdAdd />}</button>{openModal && <PlansModal closeModal={setOpenModal} />}</h3>
+            <div class="table-batch-list">
+                <table className="table batch-list">
+                    <thead>
+                        <tr>
+                            <th>#</th>
+                            <th>Plan Name</th>
+                            <th>Description</th>
+                            <th>Status</th>
+                            <th>Price</th>
+                            <th>Tag</th>
+                            <th>Type</th>
+                            <th>Duration</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+
+                        {
+                            // plans?.map((item, index) => {
+                            //     return (
+                            //         <tr>
```

Comment:
why are these lines commented ?remove the code if not used

Reply:
These line of code is for the listing plans. For now their is no Single API to list all the plans. Once this API is implemented in back-end. the code comments can be removed. All other changes I have been made